gc/base/ParallelDispatcher.cpp

```diff
@@ -258,7 +259,8 @@ MM_ParallelDispatcher::initialize(MM_EnvironmentBase *env)
 {
 	OMR::GC::Forge *forge = env->getForge();
 
-	_threadCountMaximum = env->getExtensions()->gcThreadCount;
+	_threadCountMaximum = _threadCountAtGCInit = env->getExtensions()->gcThreadCount;
```

Comment:
break this into separate lines

I'd give _threadCountAtGCInit  a more specific name like
_threadCountMaximumAtSnapshotStartup
and surround it withe CRIU #ifdef 
(I can see that variable is used in a trace point - not sure if generation of tracepoints is aware of #ifdefs)